addons/l10n_ar/models/account_chart_template.py

```diff
@@ -27,9 +27,10 @@ def _load(self, template_code, company, install_demo):
         """
         coa_responsibility = self._get_ar_responsibility_match(template_code)
         if coa_responsibility:
+            if self._context.get('post_install_load_template'):
+                company = self.env['res.company'].browse(company.id)
             company.write({
                 'l10n_ar_afip_responsibility_type_id': coa_responsibility.id,
-                'country_id': self.env['res.country'].search([('code', '=', 'AR')]).id,
                 'tax_calculation_rounding_method': 'round_globally',
             })
```

Comment:
In relation to this PR, whenever you try to create a `res.company` for Argentina, it triggers the installation of `l10n_ar` which adds new fields on `res.company`, but since the object of company is a pre-installation of `l10n_ar` when the fields were not present the traceback we get:
```py
  File "/home/odoo/odoo/community/odoo/models.py", line 4448, in write
    raise ValueError("Invalid field %r on model %r" % (fname, self._name))
ValueError: Invalid field 'l10n_ar_afip_responsibility_type_id' on model 'res.company'
```
But when you insert a breakpoint and try to check with `fields_get` the `l10n_ar_afip_responsibility_type_id` field is present for `res.company` the object doesn't updates the fields post installation, I found a hack for this as of now.
I would like to get suggestion from framework team (@rco-odoo and @sleepy-monax )

Note: I tried browsing the object in `account/models/company.py` after post installation in `install_l10n_modules` but it didn't work

Reply:
Update:
I guess the traceback still persists because, even after the context I m passing on it and after that one more time there is call on the method without the context (at least not from my calls) but when you run the code without condition and browse the company each time the method is called, Traceback no longer persists.

I'm not sure how the ORM should work here?? And we don't want to browse the company object everytime when the method gets called